shell/common/input_events_unittests.cc

```diff
@@ -0,0 +1,227 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/common/shell_test.h"
+#include "flutter/testing/testing.h"
+
+namespace flutter {
+namespace testing {
+
+using Generator = std::function<int(int)>;
```

Comment:
I realize the units dont matter, but can you make these be points anyway? Makes this clearer to read than just an int.

Reply:
Done.

Reply:
Did you forget to push this? I still see these as `int`s. In any case, it was a nit.

Reply:
Ah, I probably misunderstood your previous comment and only added a comment. Now I've used `UnitlessTime` to make it more clear.